packages/manager/apps/web/client/app/domain/list/list-domain-layout.controller.js

```diff
@@ -93,6 +78,82 @@ export default class ListDomainLayoutCtrl extends ListLayoutHelper.ListLayoutCtr
     };
   }
 
+  loadPage() {
+    const mockData = {
+      /* LEGACY FIELDS */
+      dnssecSupported: true, // DEPRECATED: replaced with dnsState
+      domain: 'string',
+      glueRecordIpv6Supported: true,
+      glueRecordMultiIpSupported: true,
+      hostSupported: true,
+      iam: {
+        displayName: 'string',
+        id: '3fa85f64-5717-4562-b3fc-2c963f66afa6',
+        tags: ['string'],
+        urn: 'string',
+      },
+      lastUpdate: '2023-08-24T09:59:15+02:00',
+      nameServerType: 'hosted', // DEPRECATED: replaced with nameServers
+      offer: 'gold',
+      owoSupported: true,
+      parentService: {
+        name: 'string',
+        type: 'string',
+      },
+      state: 'ok', // ok, expired, restorable, deleted, pending_delete, pending_outgoing_transfer, dispute, pending_installation,
+      // NEW: pending_create, pending_incoming_transfer, technical_suspended, registry_suspended
+      suspensionState: 'not_suspended', // not_suspended, suspended
+      transferLockStatus: 'locked', // locked, locking, unavailable, unlocked, unlocking
+      whoisOwner: '25091873', // DEPRECATED: replaced with contactOwner
+
+      /* NEW FIELDS */
+      dnssecState: 'enabled', // NEW: enabled, disabled, not_supported
+
+      renewalState: 'automatic_renew', // NEW: manual_renew, automatic_renew, cancellation_requested, cancellation_complete, unpaid, expired
+      expirationDate: '2023-12-24T01:00:00+01:00', // NEW: RFC3339
+      renewalDate: '2023-12-01T01:00:00+01:00', // NEW: RFC3339. Null if cancellation_requested or no expirationDate, equals expirationDate if manual_renew, or effective renewal date if automatic_renew
+
+      contactOwner: {
+        // NEW
+        id: '25091873', // Xander contact ID
+      },
+      contactAdmin: {
+        // NEW
+        id: 'dd220470-ovh', // nichandle, may become a Xander contact ID in the future
+      },
+      contactTech: {
+        // NEW
+        id: 'dd220470-ovh', // nichandle, may become a Xander contact ID in the future
+      },
+      contactBilling: {
+        // NEW
+        id: 'dd220470-ovh', // nichandle, may become a Xander contact ID in the future
+      },
+
+      nameServers: [
+        // NEW
+        {
+          id: 1,
+          type: 'hosting', // hosting, anycast, parking, dedicated, external
+          name: 'dns13.ovh.net',
+          ipv4: '213.251.188.132', // optional
+          ipv6: '2001:41d0:1:4a84::1', // optional
+        },
+        {
+          id: 2,
+          type: 'hosting',
+          name: 'ns13.ovh.net',
+        },
+      ],
+    };
+    return this.$q.resolve({
+      data: [mockData].concat(get(this.resources, 'data')),
+      meta: {
+        totalCount: this.paginationTotalCount,
+      },
+    });
```

Comment:
**question:** Why mock data? Dont we have API?

Reply:
Not yet